extensions/vscode/webviews/homeView/src/components/EvenEasierDeploy.vue

```diff
@@ -410,7 +410,7 @@ const lastStatusDescription = computed(() => {
       ? "Not Yet Updated"
       : "Not Yet Deployed";
   }
-  if (isAbortedContentRecord.value) {
+  if (isDismissedContentRecord.value) {
```

Comment:
Should we move this check above the `isPreContentRecord` check in this method? To avoid the situation where you cancel a deploy prior to getting an ID and the status showing "Not Yet Deployed" even though it has been dismissed.

Reply:
Yes, in fact, it should go above the `deploymentError` check. This pointed out that we should be resetting the error when we set the `DismissedAt` attribute since we are not resetting the entire deployment object at the beginning of the deployment.